plugins/flatpak/gs-flatpak.c

```diff
@@ -1679,45 +1678,6 @@ gs_flatpak_refresh (GsFlatpak *self,
 			return FALSE;
 	}
 
-	/* no longer interesting */
-	if ((flags & GS_PLUGIN_REFRESH_FLAGS_PAYLOAD) == 0)
-		return TRUE;
```

Comment:
Is this patch also intended for upstream adoption? If not, is there anything that would actually call into this method with the refresh-payload flag? In other words I'd like to avoid carrying a downstream-only patch to remove this code if it would never be hit in our configuration.

Reply:
The patch will not yet be added to upstream ATM because the strategy for auto updates there is still under discussion.
I would still like to keep this change because, even if it's not hit by our configuration, I think that downloading updates during refresh is not the correct approach and (equally important) I'd rather make sure this is never possible (otherwise we could hit this again on a future rebase, so it'd be yet another thing to care about for no good reason).
Also, I am trying to push upstream a division between `refreshing` (for metadata/catalogs/etc.) and auto downloads/updates, which should be performed on a different function.